src/Backoffice/Employees/App/Notifications/TaskAssignmentNotification.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employees\App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use Lightit\Backoffice\Tasks\Domain\Models\Task;
+
+class TaskAssignmentNotification extends Notification
+{
+    use Queueable;
+
+    public function __construct(protected Task $task)
+    {
+    }
+
+    /**
+     * @return array<int, string>
+     */
+    public function via(object $notifiable): array
+    {
+        return ['mail'];
+    }
+
+    public function toMail(object $notifiable): MailMessage
+    {
+        return (new MailMessage())
+            ->subject(
+                'Task has been assigned to you | ' . ($this->task->updated_at ? $this->task->updated_at->format(
+                    'd/m/Y'
+                ) : 'N/A')
+            )
+            ->markdown('mail.assigned-task', [
+                'task' => $this->task,
+            ]);
+    }
```

Comment:
FYI:
```suggestion
    public function toMail(Employee $notifiable): MailMessage
    {
        return (new MailMessage())
            ->subject(
                'Task has been assigned to you | ' . ($this->task->updated_at ? $this->task->updated_at->format(
                    'd/m/Y'
                ) : 'N/A')
            )
            ->markdown('mail.assigned-task', [
                'task' => $this->task,
            ]);
    }
```
Since you're sending this notification like this:
```
$task->employee->notify(new TaskAssignmentNotification($task));
```
The model you're sending the notification through becomes the `$notifiable`.